packages/data/addon/adapters/dimensions/elide.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Description: The adapter for the Elide dimension model.
+ */
+import { getOwner } from '@ember/application';
+import EmberObject from '@ember/object';
+import NaviDimensionAdapter, { DimensionColumn, DimensionFilter } from './interface';
+import { ServiceOptions } from 'navi-data/services/navi-dimension';
+import { RequestV2 } from '../facts/interface';
+import ElideFactsAdapter from '../facts/elide';
+
+export default class ElideDimensionAdapter extends EmberObject implements NaviDimensionAdapter {
+  /**
+   * @property adapter - the elide fact adapter that will send asyncQuery requests for the dimension values
+   */
+  factAdapter: ElideFactsAdapter = getOwner(this).lookup('adapter:facts/elide');
```

Comment:
```suggestion
   * The elide fact adapter that will send asyncQuery requests for the dimension values
   */
  private factAdapter: ElideFactsAdapter = getOwner(this).lookup('adapter:facts/elide');
```
I think Jon was in favor of dropping the property tag